src/main/java/com/gluonhq/substrate/util/FileOps.java

```diff
@@ -671,6 +674,28 @@ public static void downloadAndUnzip(String sourceUrl, Path destPath, String file
         }
     }
 
+    /**
+     * Shorten the Java classpath with a pathing jar
+     * @param classpath A string with the classpath of files that will be added to the
+     *                 pathing jar Class-Path attribute
+     * @return a String with the path to the created pathing jar
+     * @throws IOException
+     */
+    public static String createPathingJar(String classpath) throws IOException {
+        Objects.requireNonNull(classpath);
+        Manifest manifest = new Manifest();
+        Attributes attributes = manifest.getMainAttributes();
+        attributes.put(Attributes.Name.MANIFEST_VERSION, "1.0");
+        attributes.put(Attributes.Name.CLASS_PATH,
+                classpath.replaceAll(File.pathSeparator, " "));
+        File jarFile = File.createTempFile("classpathJar", ".jar");
```

Comment:
One small remark. Wouldn't it be better to create the temporary pathing jar file in the directory provided by `ProcessPaths.getTmpPath()`?

Reply:
I don't see any difference?
`gvm/tmp` is mainly for the native-image's `TempDirectory` flag. If we want to have it under the gvm build folder (for inspection, for instance), probably better under the `gvm/<appname>` folder where we add the launcher files?

Reply:
I was thinking about it because the build/target directory is cleaned up more regularly then the system temp directory? In any case, the file size is negligibly small, so it's not really a big issue.